.env

```diff
@@ -0,0 +1 @@
+HTTPPORT=80
```

Comment:
`APP_URL` is currently required to be set manually, unfortunately.  `.env` is a magic docker file that lets you specify default environment variables that are used in the compose file.  I tinkered around with making `APP_URL` have a default, but I couldn't get it working quickly.  I could try again, though, just to make it consistent.